recipe/meta.yaml

```diff
@@ -1,16 +1,16 @@
 {% set name = "black" %}
-{% set version = "20.8b1" %}
+{% set version = "21.4b1" %}
```

Comment:
```suggestion
{% set version = "21.4b0" %}
```

Reply:
Thanks! - You caught me in mid edit there as well.